tools/pipelines-tasks/AVDAppAttachPublish/publishAVD.ps1

```diff
@@ -0,0 +1,14 @@
+param([string]$inputJsonStr, [string]$targetDLL)
+
+Write-Output $inputJsonStr
+Write-Output $targetDLL
+
+$scriptDir = $PSScriptRoot
+
+Add-Type -Path $targetDLL
+
+#Create AppAttachKernel object
+$custObj = New-Object AppAttachKernel.AppAttachKernelController($inputJsonStr)
+$token = New-Object 'System.Threading.CancellationToken'
+
+$custObj.execute($token)
```

Comment:
This DLL is a black box to me. Can you please add comments on what it is doing, where the DLL is coming from, and where the next person can go to for more information?

Reply:
More information on using AppAttach Kernel is updated in documentation (/common/lib/Readme)